Community/Tdarr_Plugin_z80t_keep_original_date.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable */
+module.exports.dependencies = [
+	'axios',
+	'path-extra',
+	'utimes,'
+];
+
+module.exports.details = function details() {
+	return {
+		id: "Tdarr_Plugin_z80t_keep_original_date",
+		Stage: "Post-processing",
+		Name: "Keep original file dates and times after transcoding",
+		Type: "Video",
+		Operation: "",
+		Description: `This plugin copies the original file dates and times to the transcoded file \n\n`,
+		Version: "1.10",
+		Link: "https://github.com/WizADSL/Tdarr_Plugins/blob/master/Community/Tdarr_Plugin_z80t_keep_original_date.js",
+		Tags: "post-processing,dates,date",
+		Inputs: [{
+				name: 'server',
+				tooltip: `IP address or hostname of the server assigned to this node, will be used for API requests.  If you are running nodes within Docker you should use the server IP address rather than the name.
+      
+      \\nExample:\\n
+       tdarrserver
+
+      \\nExample:\\n
+       192.168.1.100`
+			}, {
+				name: 'extensions',
+				tooltip: `When files are trans-coded the file extension may change, enter a list of extensions to try and match the original file with in the database after trans-coding. Default is the list of container types from library settings. The list will be searched in order and the extension of the original file will always be checked first before the list is used.
+      
+      \\nExample:\\n
+       mkv,mp4,avi`
+			},
+			{
+				name: 'log',
+				tooltip: `Write log entries to console.log. Default is false.
+      
+      \\nExample:\\n
+       true`
+			},
+		],
+	};
+};
+
+module.exports.plugin = async function plugin(file, librarySettings, inputs) {
+
+	var responseData = {
+		file,
+		removeFromDB: false,
+		updateDB: false,
+		infoLog: '',
+	};
+
+	try {
+
+		if (inputs.server == undefined || inputs.server.trim() === '') {
+			responseData.infoLog += 'Tdarr server name/IP not configured in library transcode options\n';
+			return responseData;
+		}
+
+		var fs = require("fs");
+		var path = require("path");
+		if (fs.existsSync(path.join(process.cwd(), "/npm"))) {
+			var rootModules = path.join(process.cwd(), "/npm/node_modules/");
+		} else {
+			var rootModules = "";
+		}
+
+		var axios = require(rootModules + "axios");
+		var utimes = require(rootModules + "utimes");
+
+		log('Waiting 5 seconds...');
+
+		var extensions = inputs.extensions;
+		if (extensions == undefined || extensions.trim() === '') {
+			extensions = librarySettings.containerFilter;
+		}
+		extensions = extensions.split(',');
+
+		await new Promise(resolve => setTimeout(resolve, 5000));
+		var response = await getFileData(file._id, extensions, inputs.server);
+
+		//log('Got response for ' + file._id);
```

Comment:
Could you remove all the commented out lines of code though?

Reply:
Done.